semver/__main__.py

```diff
@@ -0,0 +1,3 @@
+from semver.cli import main
```

Comment:
Another idea, not sure if this is really needed.

The `wheel` package allows to "execute" the wheel whl file itself. Sounds strange, but it's true. :wink: To support that, our semver `__main__.py` needs to be adapted like this:

```python
import sys

def main():  # needed for console script
    if __package__ == '':
        import os.path
        path = os.path.dirname(os.path.dirname(__file__))
        sys.path[0:0] = [path]
    import semver.cli
    sys.exit(semver.cli.main())


if __name__ == "__main__":
    sys.exit(main())
```

With that change, it's possible to "run" the wheel like this:

```
$ python3 semver-3.0.0*.whl/semver -h
usage: semver [-h] [--version] {compare,bump,check,nextver} ...

[... help output pruned ...]
```

Reply:
That's peculiar, I've never heard of running wheels before. I wonder why one would want to do that. Anyway, doesn't hurt to integrate it.

Reply:
Hehe, yes it is. But I think it's not too hard to add it. It may have some benefits as you don't need to install the wheel on the system and can just run it.
We can add that into our documentation or we can leave that as an "easter egg". :wink: